supervisor/backups/backup.py

```
@@ -334,6 +335,35 @@
         data = padder.update(decrypt.update(b64decode(data))) + padder.finalize()
         return data.decode()
 
+    async def validate_password(self) -> bool:
+        """Validate backup password"""
```

Ruff (0.8.2)

First line should end with a period

Add period

(D400)